experimental/sev_guest/src/ghcb.rs

```diff
@@ -91,6 +91,13 @@ pub struct Ghcb {
     pub ghcb_usage: u32,
 }
 
+impl Default for Ghcb {
+    fn default() -> Self {
+        // This is safe because all-zeroes is a valid value of type Ghcb.
+        unsafe { core::mem::zeroed() }
```

Comment:
Optional: because FromBytes is already derived for Ghcb, you could also use `new_zeroed`, althout that is just a wrapper for the exact same code, so probably does not make a difference. (https://docs.rs/zerocopy/latest/src/zerocopy/lib.rs.html#228-237)

Reply:
Ah, I missed that. I think `new_zeroed` is a good idea here as this means we don't have to resort to explicit unsafe code.